pkgs/applications/misc/ideamaker/default.nix

```diff
@@ -0,0 +1,147 @@
+{
+  stdenv,
+  autoPatchelfHook,
+  curl,
+  dpkg,
+  fetchurl,
+  fetchzip,
+  lib,
+  libcork,
+  libGLU,
+  makeDesktopItem,
+  openssl_1_1,
+  qtbase,
+  qtserialport,
+  quazip,
+  shared-mime-info,
+  wrapQtAppsHook,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ideamaker";
+  version = "4.3.3";
+  buildNumber = "6560";
+  src = fetchurl {
+    url = "https://download.raise3d.com/${finalAttrs.pname}/release/${finalAttrs.version}/ideaMaker_${finalAttrs.version}.${finalAttrs.buildNumber}-ubuntu_amd64.deb";
+    sha256 = "sha256-aTVWCTgnVKD16uhJUVz0vR7KPGJqCVj0xoL53Qi3IKM=";
+  };
+
+  openssl_1_0_1 = openssl_1_1.overrideAttrs (previous: {
+    version = "1.0.1u";
+    src = fetchurl {
+      url = "https://www.openssl.org/source/openssl-1.0.1u.tar.gz";
+      sha256 = "0fb7y9pwbd76pgzd7xzqfrzibmc0vf03sl07f34z5dhm2b5b84j3";
+    };
+    patches = [ ];
+    withDocs = false;
+    outputs = lib.lists.remove "doc" previous.outputs;
+    meta.knownVulnerabilities = [
+      "OpenSSL 1.0.1 is reaching its end of life on 2016/12/31 https://endoflife.software/applications/security-libraries/openssl"
+    ];
```

Comment:
At a minimum, this needs to be reworded b/c 2016/12/31 was almost 8 years ago (aside: time is cruel), but I think you also need to see if there are known CVEs against this version to link here.